### PR TITLE
fix: fix Airlines class declaration

### PR DESCRIPTION
**Pull request by andrejak**
This fixes the 2nd issue mentioned in https://github.com/duffelhq/duffel-api-javascript/issues/797. 

We were getting this error, only on the Airlines class:

```
node_modules/@duffel/api/dist/typings.d.ts:3921:12 - error TS1038: A 'declare' modifier cannot be used in an already ambient context.

3921     export declare class Airlines extends Resource {
```

I tried comparing the Airlines file to other similar ones (eg Airports). Making a small change in the doc comment about Airlines helped rebuild the typings file correctly on my machine so I'm hoping/assuming that will happen on the built one too...

Screenshot 2023-10-13 at 10 17 55

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,8 +51,7 @@
     "ansi-regex": "^5.0.1",
     "set-value": "^4.0.1",
     "minimist": "1.2.8",
-    "semver": "^7.5.3",
-    "word-wrap": "npm:@aashutoshrathi/word-wrap"
```
Comment (andrejak) [inline package.json:55]: This is unrelated, just a small fix I wanted to do anyway (the regular word-wrap has security vulnerabilities fixes in now so we don't need to rely on this version anymore)

**Review by igorp1 — APPROVED** [review]
